packages/driver/cypress/integration/e2e/multi-domain/multi_domain_uncaught_errors_spec.ts

```diff
@@ -240,21 +257,220 @@ describe('multi-domain - uncaught errors', { experimentalSessionSupport: true },
     })
   })
 
-  it('does not fail if thrown custom error has a readonly name', (done) => {
-    cy.once('fail', (err) => {
-      expect(err.name).to.include('CustomError')
-      expect(err.message).to.include('custom error')
+  describe('unserializable errors', () => {
+    it('handles users throwing dom elements', (done) => {
+      cy.on('fail', (err) => {
+        expect(err.name).to.equal('CypressError')
+        expect(err.message).to.equal('`cy.switchToDomain()` could not serialize or map the thrown value. Please make sure the value being thrown is supported by the structured clone algorithm.')
+        done()
+      })
+
+      // @ts-ignore
+      cy.switchToDomain('foobar.com', () => {
+        throw document.createElement('h1')
+      })
+    })
+
+    it('handles users throwing functions', (done) => {
+      cy.on('fail', (err) => {
+        expect(err.name).to.equal('CypressError')
+        expect(err.message).to.equal('`cy.switchToDomain()` could not serialize or map the thrown value. Please make sure the value being thrown is supported by the structured clone algorithm.')
+        done()
+      })
+
+      // @ts-ignore
+      cy.switchToDomain('foobar.com', () => {
+        throw () => undefined
+      })
+    })
+
+    it('handles users throwing symbols', (done) => {
+      cy.on('fail', (err) => {
+        expect(err.name).to.equal('CypressError')
+        expect(err.message).to.equal('`cy.switchToDomain()` could not serialize or map the thrown value. Please make sure the value being thrown is supported by the structured clone algorithm.')
+        done()
+      })
+
+      // @ts-ignore
+      cy.switchToDomain('foobar.com', () => {
+        throw Symbol('foo')
+      })
+    })
+
+    it('handles users throwing promises', (done) => {
+      cy.on('fail', (err) => {
+        expect(err.name).to.equal('CypressError')
+        expect(err.message).to.equal('`cy.switchToDomain()` could not serialize or map the thrown value. Please make sure the value being thrown is supported by the structured clone algorithm.')
```

Comment:
I wonder if something like this makes more sense
```suggestion
        expect(err.message).to.equal('`cy.switchToDomain()` could not determine the thrown value. Please make sure the value being thrown is supported by the structured clone algorithm.')
```

Reply:
I think just `serialize` is fine. Users will have to be familiar with the serialization limits when they're using the multi-domain API.

Reply:
@chrisbreiding, are you saying to remove `or map` and just have `serialize`?

Reply:
Yeah, I'd say remove `or map`. I understand the intent behind it, but think that will just be confusing to users.

Reply:
`or map` removed in c6648651f1